pkg/sippyserver/server.go

```diff
@@ -724,6 +727,21 @@ func (s *Server) jsonComponentReportFromBigQuery(w http.ResponseWriter, req *htt
 		failureResponse(w, http.StatusInternalServerError, fmt.Sprintf("error querying component from big query: %v", errs))
 		return
 	}
+
+	// Iterate through all RegressedTests in outputs.Rows and inject HATEOAS links for regressions
+
+	for i := range outputs.Rows {
```

Comment:
I might have missed something here. I thought the middleware is already do this, no?